src/Microsoft.VisualStudio.Services.Agent/ThrottlingReportHandler.cs

```diff
@@ -45,13 +45,16 @@ protected async override Task<HttpResponseMessage> SendAsync(HttpRequestMessage
             // Inspect whether response has throttling information
             IEnumerable<string> vssRequestDelayed = null;
             IEnumerable<string> vssRequestQuotaReset = null;
+
             if (response.Headers.TryGetValues(HttpHeaders.VssRateLimitDelay, out vssRequestDelayed) &&
                 response.Headers.TryGetValues(HttpHeaders.VssRateLimitReset, out vssRequestQuotaReset) &&
                 !string.IsNullOrEmpty(vssRequestDelayed.FirstOrDefault()) &&
                 !string.IsNullOrEmpty(vssRequestQuotaReset.FirstOrDefault()))
             {
-                TimeSpan delay = TimeSpan.FromMilliseconds(int.Parse(vssRequestDelayed.First()));
-                DateTime expiration = DateTime.Parse(vssRequestQuotaReset.First());
+                TimeSpan delay = TimeSpan.FromMilliseconds(double.Parse(vssRequestDelayed.First()));
+                int expirationEpoch = int.Parse(vssRequestQuotaReset.First());
```

Comment:
It appears that `VssRateLimitReset` used to be a `DateTime` value, but now it's an `int` representing an epoch. Fixed the parsing so it doesn't throw an exception.

Reply:
is there any way to not hard code `1970, 1, 1, 0, 0, 0`

Reply:
Let me see if I can find a constant that represents the same datetime.